apollo/server/datasources/executive.ts

```diff
@@ -85,10 +85,10 @@ export default class ExecutiveAPI extends DataSource<ContextBase> {
   }
 
   /**
-   * Update a new executive
+   * Update an executive
    * @async
    * @param {ExecutiveCreationAttributes} arg - The arg for the executive
-   * @returns {Promise<ExecutiveAttributes>} Number of people updated and an instance of the updated executive
+   * @returns {Promise<ExecutiveAttributes>} Number of executives updated and instances of the updated executive
```

Comment:
wrong type, can remove it, typedoc can infer if from ts
`the updated executive` => `updated executives`